### DISCUSSION TITLE
HyperV support for Podman machine

### DISCUSSION THREAD

**Discussion by baude** [containers/podman#17612] [Ideas]
I am working on HyperV support for Podman machine and would like to identify community members who might be willing to help test.  I will provide a binary and the bootable image.  My expectation is you give it a try and report back whether it worked or not.

If this is something you are interested in testing, please give comment a thumbs up so it when I am ready for testing, it will prompt you.

Thank you so very much.

**Comment by rchaganti** [comment]
Count me in!

**Comment by sumantro93** [comment]
Count me in too!

**Comment by jeffmaury** [comment]
Same for me

**Comment by Starimmigrant** [comment]
Count me in too

**Comment by baude** [comment]
Hey folks, if you are interested in hyperv, checkout out -> https://fedorapeople.org/groups/podman/testing/hyperv/. ... be sure to read the readme in there.
